solarwinds_apm/distro.py

```diff
@@ -298,15 +293,13 @@ def load_instrumentor(self, entry_point: EntryPoint, **kwargs):
                     kwargs,
                 )
 
-        # If SW_APM_EXPORT_(METRICS|LOGS)_ENABLED is set to false,
+        # If SW_APM_EXPORT_METRICS_ENABLED is set to false,
         # then we load the instrumentor with NoOp providers to disable
-        # Otel instrumentor-based metrics/logs export if supported.
+        # Otel instrumentor-based metrics export if supported.
         # Assumes kwargs are ignored if not implemented
         # for the current instrumentation library.
         if self._instrumentor_metrics_enabled is False:
             kwargs["meter_provider"] = NoOpMeterProvider()
-        if self._instrumentor_logs_enabled is False:
-            kwargs["logger_provider"] = NoOpLoggerProvider()
```

Comment:
No upstream instrumentor supports this kwarg at the moment, so this does nothing.